### DISCUSSION TITLE
Welcome to ShopifyCheckoutJS Discussions!

### DISCUSSION THREAD

**Discussion by adearriba** [adearriba/ShopifyCheckoutJS#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VladimirBlagojevic** [comment]
Great library, tnx for all your hard work.
Do you maybe have a full example of implementation to some Shopify theme checkout, with a few custom fields
